### PR TITLE
dashboard.demo v 0.1.0

### PR DESCRIPTION
**Pull request by rahulchauhan049**

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+^.*\.Rproj$
+^\.Rproj\.user$
+^data-raw$
+dev_history.R
+^dev$
+$run_dev.*
+^CODE_OF_CONDUCT\.md$
+^inst\app\www\img
```
Comment (thiloshon) [inline .Rbuildignore]: Why are you ignoring a file inside inst? By default it should be ignored right?

File: R/app_ui.R [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+#' @import bdvis
+#' @import ggplot2
+#' @import tidyr
+#' @import treemap
+#' @import sunburstR
+#' @import formattable
+#' @import summarytools
+#' @import leaflet.extras
+#' @import sp
+#' @import bddwc.app
```
Comment (thiloshon) [inline R/app_ui.R]: Add all of this imports in single line. or few multiple lines.

File: DESCRIPTION [hunk 3]
```diff
@@ -0,0 +1,53 @@
+Package: dashboard.demo
+Title: Dashboard For biodiversity data visualization
+Version: 0.1.0
+Date: - 25-08-2019
```
Comment (thiloshon) [inline DESCRIPTION]: Why is there a - before date?

File: DESCRIPTION [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+Author-s@R: c(
```
Comment (thiloshon) [inline DESCRIPTION]: Make it `Authors@R:`

File: R/mod_dataSummary.R [hunk 5]
```diff
@@ -0,0 +1,585 @@
...
+      )
+    )
+    
+    shinydashboard::infoBox(
+      "# of Geo Coordinates",
+      if(latitude>longitude){
+        longitude
+      }else{
+        latitude
+      },
```
Comment (thiloshon) [inline R/mod_dataSummary.R:396]: this blocks needs to be reformatted

File: R/mod_dataSummary.R [hunk 6]
```diff
@@ -0,0 +1,585 @@
...
+            dataset()["species"]
+          )
+        )
+      ),
+      icon = icon("stripe-s"),
+      color = "teal",
+      width = 4
+    )
+  })
+}
```
Comment (thiloshon) [inline R/mod_dataSummary.R]: Add an empty line at the end of every files

**Review by thiloshon — APPROVED** [review]
Please fix the small suggestions, I'll merge then.

File: .Rbuildignore (hunk 1 above)
Comment (rahulchauhan049) [reply .Rbuildignore]: actually, they gifs that are used by readme.md. So no need of them in R build so I ignore them.

File: DESCRIPTION (hunk 3 above)
Comment (rahulchauhan049) [reply DESCRIPTION]: typing error

**Pull request merged**